Resources/views/Collector/swiftmailer.html.twig

```diff
@@ -79,8 +177,11 @@
                                 {{- message.body|e('html') }}
                             {%- endif -%}
                         </pre>
-                        {% for messagePart in message.children if messagePart.contentType == 'text/plain' or messagePart.contentType == 'text/html' %}
-                            <h4>Alternative part</h4>
+                    </div>
+
+                    {% for messagePart in message.children if messagePart.contentType == 'text/plain' or messagePart.contentType == 'text/html' %}
+                        <div class="card-block">
+                            <span class="label">Alternative part ({{ messagePart.contentType }})</span>
                             <pre>
                                 {%- if messagePart.charset %}
                                     {{- messagePart.body|e('html', messagePart.charset)|convert_encoding('UTF-8', messagePart.charset) }}
```

Comment:
I think we have a bug here: we are double-escaping it as ``convert_encoding`` is not marked as safe (which is logical) or as preserving safety.
The following would be better (relying on the auto-escaping for escaping):

```twig
{{- messagePart.body|convert_encoding('UTF-8', messagePart.charset) }}
```

The same apply a few lines above for the message body btw